ManuProjectFiles/app/src/test/java/com/example/manu/ExampleUnitTest.kt

```diff
@@ -285,5 +285,29 @@ class ExampleUnitTest {
         }
     }
 
+    @Test
+    fun noDuplicatesInBirdDatabase(){
+        var stillValid = true
+        BirdDatabase.compileDatabase()
+
+        val birds = BirdDatabase.getBirdList()
+        val birds2 = BirdDatabase.getBirdList()
+
+        var birdTrackerList2 = 0
+
+        for ((birdTrackerList1, bird) in birds.withIndex()){
+            for ((birdTrackerList2, secondBird) in birds2.withIndex()){
+                if(birdTrackerList1 != birdTrackerList2) {
+                    if (bird.getBirdName() == secondBird.getBirdName()) {
+                        stillValid = false
+                    }
+                }
+            }
+        }
+
+        assertEquals(true, stillValid)
```

Comment:
A reminder from our previous bug fix to ensure you check it is true when it should be true or false when it should be false. For example, if a false value flags that there should be no issue, then checking that it equals false is correct.